packages/Popover/styles.js

```diff
@@ -7,9 +7,10 @@ export const Arrow = styled(PopoverArrow)`
   color: ${th('popovers.default.backgroundColor')};
 `
 
-export const ArrowItem = styled('svg')(
+export const ArrowItem = styled.svg(
```

Comment:
Pour info, tu peux préfixer les éléments html par `Box` pour pas avoir a ajouter `system`, e.g:
```suggestion
export const ArrowItem = styled.svgBox(
```